appveyor.yml

```diff
@@ -3,39 +3,22 @@ branches:
   only:
     - master
 
-image: Visual Studio 2015
+image: Visual Studio 2019
 
 environment:
   matrix:
-    - platform: win32
-      ADDR_MODEL: 32
-      ARCH: win32-msvc9
-      MSVCVERSION: 9.0
-      PYTHONPATH: c:\Python27\
-      PY_VER: 27
-      BOOST_CFG: >-
-          using python : 2.7 : c:/python27/python.exe : c:/python27/include : c:/python27/libs ;
-    - platform: x64
-      ADDR_MODEL: 64
-      ARCH: x64-msvc9
-      MSVCVERSION: 9.0
-      PYTHONPATH: c:\Python27-x64\
-      PY_VER: 27
-      BOOST_CFG: >-
-          using python : 2.7 : c:/python27-x64/python.exe : c:/python27-x64/include : c:/python27-x64/libs ;
```

Comment:
This will slightly complicate the AppVeyor script in PyTango, since it assumes the same version of boost for all Python versions, but not too much of a problem.  Python 2.7 is supposed to be EOL... 🙂

Reply:
Might be a rushed decision on my side to remove Python 2.7, let me know if you want me to re-enable it.

Reply:
If the build still works, then I suggest we leave it in.  PyTango will have to supported 2.7 for a long time still.

Reply:
I'll have to revert the image to `Visual Studio 2015` since the 2019 one does not have Visual Studio 2010 required to build for Python 2.7. Unless we split the build in two branches to create two pipelines with different images.

Reply:
No, that sounds like too much work here.  Let us keep what you have.